app/Models/Publisher.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Publisher extends Model
+{
+    /**
+     * Get the Publisher
+     */
+    public function __construct()
+    {
+
+    }
+
+    public function game() {
```

Comment:
this should be `games` because a publisher can have many games

Reply:
resolved!